### PR TITLE
Few python3 related fixes

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

**Comment by fruch** [comment]
running now on BYO, for making sure no regression on python2 code:
https://jenkins.scylladb.com/job/scylla-master/job/manual-and-scheduled-tests/job/byo_build_tests_dtest/700/

**Comment by fruch** [comment]
@hagitsegev @yarongilor, those fixes would also address the issue failing the python matrix

File: ccmlib/common.py [hunk 1]
```diff
@@ -366,8 +366,14 @@ def isDse(install_dir):
 
 
 def isScylla(install_dir):
+    scylla_version = os.environ.get('SCYLLA_VERSION', None)
```
Comment (bhalevy) [inline ccmlib/common.py]: 1. Is this python3 related fix?  Seems orthogonal and so should be done in a separate patch.

2. why do it unconditionally?
can't it be done only `if install_dir is None:`

File: ccmlib/scylla_node.py [hunk 2]
```diff
@@ -374,7 +374,6 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
                 except Exception as msg:
                     print("{}. Looking for offending processes...".format(msg))
                     for proc in psutil.process_iter():
-                        print  proc.cmdline()
```
Comment (bhalevy) [inline ccmlib/scylla_node.py]: python3 related?

Comment (fruch) [reply ccmlib/scylla_node.py]: yes, it's breaks python, and it's not actully need (I think it's my mistake in older PR)

File: ccmlib/common.py (hunk 1 above)
Comment (fruch) [reply ccmlib/common.py]: as for 1, yes it can be separated into it's own commit

Comment (fruch) [reply ccmlib/common.py]: as for 2, yes we could put it into the right context

Comment (fruch) [reply ccmlib/common.py]: opened
https://github.com/scylladb/scylla-ccm/pull/189
for this one
